contracts/HybridCustody.cdc

```diff
@@ -732,7 +736,9 @@ pub contract HybridCustody {
             self.acctOwner = to
             self.relinquishedOwnership = false
 
-            emit OwnershipGranted(id: self.uuid, child: self.acct.address, owner: to)
+            emit OwnershipUpdated(id: self.uuid, child: self.acct.address, owner: to, active: false)
```

Comment:
The concern I have here is that `giveOwnership()` == `removeOwned()` whereas before these actions were distinct.